ibis/backends/base/sqlglot/compiler.py

```diff
@@ -822,6 +822,14 @@ def visit_Window(self, op, *, how, func, start, end, group_by, order_by):
         end_value = end.get("value", "UNBOUNDED")
         end_side = end.get("side", "FOLLOWING")
 
+        if f"{start_value} {start_side}".lower() in ("0 following", "0 preceding"):
```

Comment:
Could `if start_value == 0` be sufficient?

Reply:
I was able to simplify it to `getattr(start_value, "this", None) == "0"`. `start_value` is a sqlglot object, so I have to handle that.